modules/default.nix

```
@@ -0,0 +1,9 @@
+{ ... }:
+
+{
+  imports = [
+    ./soxin
+
+    ./workstation/bluetooth.nix
```

```suggestion
    ./hardware/bluetooth.nix
```

Let's try to follow the categories on NixOS side and not invent our own. Shabka is pretty messy :disappointed: